chapters/3-package-information.md

```diff
@@ -6,12 +6,12 @@ Cardinality: Optional, one or many.
 
 In `tag:value` format, the order in which package and files occur is syntactically significant.
 
-A new Package Information section is denoted by the [Package Name](#3.1) field.
-All Package Information fields must be grouped together before the start of a [Files section](4-file-information.md), if file(s) are present.
-All files contained in a package must immediately follow the applicable Package Information.
-A new Package Information section (via Package Name) denotes the start of another package.
-Sub-packages should not be nested inside a Package Information section, but should be separate and should use a Relationship to clarify.
-Annotations and Relationships for the package may appear after the Package Information before any file information.
+* A new Package Information section is denoted by the [Package Name](#3.1) field.
```

Comment:
Here (and in the similar bits below for Files and Snippets), 2.1 had bullet points -- reinserted here so it doesn't collapse down into one massive paragraph